internal/pkg/cli/cmd/template/describe.go

```diff
@@ -7,19 +7,56 @@ import (
 
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/helpmsg"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	describeOp "github.com/keboola/keboola-as-code/pkg/lib/operation/template/local/repository/describe"
 )
 
-func DescribeCommand(_ dependencies.Provider) *cobra.Command {
+func DescribeCommand(p dependencies.Provider) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "describe",
+		Use:   "describe <template> <version>",
```

Comment:
ak je version optional , tak by to mohlo byt `describe template [version]`

Podobne ako tu:
https://github.com/keboola/keboola-as-code/blob/cce0e05a3dc1483afcb22b8e659a46e051ebd814/internal/pkg/cli/cmd/sync/push.go#L16

(tam su este aj `"`, kedze su to 2 slova)

Reply:
jo, koukám že placeholder se většinou dává do <> a optional parametr do [], tak imho `describe <template> [version]` by mohlo být ok a je to konzistentní s tím pushem

Reply:
:+1: